pkgs/applications/misc/navi/default.nix

```diff
@@ -0,0 +1,36 @@
+{ rustPlatform, fetchFromGitHub, stdenv, fzf, makeWrapper, ... }:
```

Comment:
Might as well take `lib` as an argument directly instead of `stdenv`, since that’s all it’s used for.